hummingbot/core/utils/exchange_rate_conversion.py

```diff
@@ -167,7 +172,8 @@ def convert_token_value(self, amount: float, from_currency: str, to_currency: st
     def start(self):
         self.stop()
         for data_feed in self._data_feeds:
-            data_feed.start()
+            if not data_feed.started:
```

Comment:
You should move all the network related stuff here to `start_network()`, and also ensure they're stopped in `stop_network()`

Reply:
I made the data feed into network modules but not the Exchange rate module, so Exchange rate module will need to start the data feed if they are not started yet.